internal/webhook/external/receiver.go

```diff
@@ -71,6 +74,11 @@ func (b *baseWebhookReceiver) GetDetails() kargoapi.WebhookReceiverDetails {
 	return b.details
 }
 
+// getMaxRequestBodyBytes
```

Comment:
unfinished comment?

Reply:
Fixed in 10b5b844e7ea72dcd066a046195c8215929d8518